src/AppBundle/Entity/PostAddress.php

```diff
@@ -180,12 +180,12 @@ public function getGeocodableAddress(): string
         return (string) GeocodableAddress::createFromAddress($this);
     }
 
-    public function equals(self $other)
+    public function equals(self $other): bool
     {
-        return $this->address !== $other->getAddress()
-               || $this->cityName !== $other->getCityName()
-               || $this->postalCode !== $other->getPostalCode()
-               || $this->country !== $other->getCountry()
+        return mb_strtolower($this->address) !== mb_strtolower($other->getAddress())
```

Comment:
Yeah, wrong copy-paste... I've fixed when adding the missing tests ;) See 7aaff81.